shimmingtoolbox/cli/b1shim.py

```diff
@@ -28,14 +29,17 @@
                    "3 - Maximizes the minimum B1+ value for better efficiency.")
 @click.option('--target', 'target', type=float, required=False,
               help="Target B1+ value used by algorithm 2 in nT/V")
-@click.option('--sed', 'sed', type=float, required=False,
+@click.option('--vop', 'fname_vop', type=click.Path(exists=True), required=False,
+              help="Path to the SarDataUser.mat file containing the VOP used for SAR constraint. This file can be "
+                   "found on the scanner in C:/MedcomMriProduct/PhysConfig.")
+@click.option('--sed', 'sed', type=float, required=False, default=1.5,
               help="Factor (=> 1) to which the local SAR after optimization can exceed the CP mode local SAR."
                    "SED between 1 and 1.5 usually work with Siemens scanners. Higher SED allows more liberty for RF"
                    "shimming but might result in SAR excess at the scanner.")
 @click.option('-o', '--output', 'path_output', type=click.Path(),
               default=os.path.join(os.curdir, 'b1_shim_results'), show_default=True,
               help="Directory to output shim weights text file and figures.")
-def b1shim_cli(fname_b1_map, fname_mask, fname_cp_weights=None, algorithm=1, target=None, q_matrix=None, sed=1.5,
+def b1shim_cli(fname_b1_map, fname_mask, fname_cp_weights=None, algorithm=1, target=None, fname_vop=None, sed=1.5,
```

Comment:
You should not put default values since they are already defined in the click decorator.

Reply:
Corrected in [b4aa255](https://github.com/shimming-toolbox/shimming-toolbox/pull/315#:~:text=up%20CLI%20arguments-,b4aa255,-Merge%20state)